bridge/client/app/_components/ktb-webhook-settings/ktb-webhook-settings.component.html

```diff
@@ -117,6 +117,30 @@ <h2>Webhook configuration</h2>
       <dt-error *ngIf="getFormControl('proxy').errors?.url">URL must start with http(s)://</dt-error>
     </dt-form-field>
   </div>
+  <div class="mt-3 mb-1" fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="15px">
+    <dt-form-field uitestid="edit-webhook-field-sendFinished">
+      <dt-checkbox formControlName="sendFinished">Send finished event</dt-checkbox>
+    </dt-form-field>
+    <div [dtOverlay]="sendFinishedOverlay" style="cursor: pointer" [dtOverlayConfig]="sendFinishedOverlayConfig">
+      <dt-icon name="information" class="info mr-1"></dt-icon>
+    </div>
+    <ng-template #sendFinishedOverlay>
+      <p>
+        If you subscribe your webhook to an event of type triggered, Keptn automatically sends a started event, executes
```

Comment:
May I ask for a change of the info message to: 


---

If you subscribe your Webhook to a task event of type triggered, the corresponding started and finished events are sent automatically. 

To move the responsibility of sending the finished event to the Webhook receiver, disable this checkbox. This is especially required for Webhooks that trigger a long-running task.